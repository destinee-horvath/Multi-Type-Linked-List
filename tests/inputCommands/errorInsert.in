INSERT 0
INSERT -1 -1 
INSERT -1 0 element
INSERT element
INSERT a b element
insert
