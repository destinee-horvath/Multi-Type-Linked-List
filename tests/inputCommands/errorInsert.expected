INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INPUT
